docs: update getting started to include WebStorm (#29845)
nuxt/nuxt: docs/1.getting-started/2.installation.md

#### Prerequisites

> - **Node.js** - [`v18.0.0`](https://nodejs.org/en) or newer

Reviewer: ```suggestion
- **Node.js** - [`18.`](https://nodejs.org/en) or newer (but we recommend the [active LTS release](https://github.com/nodejs/release#release-schedule))
```

> - **Text editor** - We recommend [Visual Studio Code](https://code.visualstudio.com/) with the [official Vue extension](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (previously known as Volar) or [WebStorm](https://www.jetbrains.com/webstorm/) and other [JetBrains IDEs](https://www.jetbrains.com/ides/), that offer great Nuxt support right out-of-the-box.

Reviewer: ```suggestion
- **Text editor** - There is no IDE requirement, but we recommend [Visual Studio Code](https://code.visualstudio.com/) with the [official Vue extension](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (previously known as Volar) or [WebStorm](https://www.jetbrains.com/webstorm/), which, along with [other JetBrains IDEs](https://www.jetbrains.com/ides/), offers great Nuxt support right out-of-the-box.
```
